src/main/java/org/bihealth/mi/easysmpc/components/DialogEmailConfig.java

```diff
@@ -49,24 +52,28 @@
 public class DialogEmailConfig extends JDialog implements ChangeListener {
 
     /** SVUID */
-    private static final long      serialVersionUID = -5892937473681272650L;
-    /** E-Mail and password entry */
-    private EntryEMailPassword     emailPasswordEntry;
-    /** E-mail server entry */
-    private EntryServers           serversEntry;
-    /** Port of e-mail servers entry */
-    private EntryServerPorts       serverPortsEntry;
+    private static final long                serialVersionUID = -5892937473681272650L;
     /** Button */
-    private JButton                buttonOK;
+    private JButton                          buttonOK;
     /** Result */
-    private ConnectionIMAPSettings result;
+    private ConnectionIMAPSettings           result;
     /** Parent frame */
-    private JFrame                 parent;
-    /** Radio button group for encryption IMAP */
-    private ComponentRadioEntry    radioEncryptionIMAP;
-    /** Radio button group for encryption STMP */
-    private ComponentRadioEntry    radioEncryptionSMTP;
-
+    private JFrame                           parent;
+    /** Radio button group dialog type simple/advanced */
+    private ComponentRadioComfirmSwitchEntry radioDialogType;
+    /** Central panel */
+    JPanel                                   central;
+    /** Init finished */
+    private boolean                          initFinished     = false;
+    /** Entry for IMAP details */
+    EntryEMailDetails                        entryIMAPDetails;
```

Comment:
Why does this (and other fields) have package visibility? Is this really necessary?